internal/hcsoci/hcsdoc_wcow.go

```diff
@@ -367,6 +368,49 @@ func createWindowsContainerDocument(ctx context.Context, coi *createOptionsInter
 		return nil, nil, err
 	}
 
+	// Setup WER registry keys for local process dump creation if specified.
+	// https://docs.microsoft.com/en-us/windows/win32/wer/collecting-user-mode-dumps
+	procDumpLoc := coi.Spec.Annotations[oci.AnnotationProcessDumpLocation]
+	if (coi.HostingSystem != nil && coi.HostingSystem.ProcessDumpLocation() != "") || procDumpLoc != "" {
+		// If a process dump path was specified at pod creation time for a hypervisor isolated pod, then
+		// use this value. If one was specified on the container creation document then override with this
+		// instead. Unlike Linux, Windows containers can set the dump path on a per container basis.
+		var dumpPath string
+		if coi.HostingSystem != nil {
```

Comment:
is this just to be consistent with linux? otherwise wouldn't it make sense to use the one provided explicitly and default to the one specified at pod creation time?
```
dumpPath := procDumpLoc
if coi.HostingSystem != nil {
    dumpPath = coi.HostingSystem.ProcessDumpLocation()
}
```

Reply:
What I was thinking is we'd default to the one on the pod spec (if present) but if one was provided on a container spec we'd just override with that, which I'm not doing here now that I'm re-reading... 

But yes, the idea was the default scenario to be in line with Linux, e.g. you could just provide the pod annotation and not have to think much afterwards, but we do allow a bit of extra customization for Windows, as the path is configurable per container.

Reply:
sounds good.